app/src/main/java/org/wikipedia/util/L10nUtil.kt

```diff
@@ -67,6 +68,22 @@ object L10nUtil {
         return localizedStrings
     }
 
+    // To be used only for plural strings and strings requiring arguements
+    fun getResourcesForWikiLang(languageCode: String): Resources? {
```

Comment:
Is it possible to combine this function with `getStringsForLocale` by adding an argument for the plural strings?

Reply:
The problem with this is the arguments @cooltey